site/source/index.md

```diff
@@ -0,0 +1,212 @@
+---
```

Comment:
This is basically the Readme in the package in Atmosphere ... that is what we are using in the readme too .. 

_I think we could add content to complete talking a little bit about the Blaze API ._ 😄 😄